include/pomerol/GreensFunction.h

```diff
@@ -50,7 +51,7 @@ class GreensFunction : public Thermal, public ComputableObject {
     /** A list of pointers to parts (every part corresponds to a part of the annihilation operator
      * and a part of the creation operator).
      */
-    std::list<GreensFunctionPart*> parts;
+    std::vector<GreensFunctionPart> parts;
```

Comment:
I noticed sometimes we use a shared pointer and some time instances. What's the rationale?

Reply:
A couple of classes were storing parts in vectors of smart pointers because I forgot to change them to plain instances.
Those are updated in the latest commit.